docs/api/python/lambda_function.rst

```diff
@@ -0,0 +1,25 @@
+Lambda Function
+====================================
+
+.. note::
+
+    Lambda Function support is an alpha and under active development.
+    Please report any bugs or let us know of any feature requests.
+
+An AWS Lambda Function is a serverless provided by AWS. It allows to run applications and backend services without
```

Comment:
In this section would be worth highlighting the two core usage paths - passing in a function callable, or the typical lambda flow where you provide paths to code and a handler function name